compass/ocean/tests/internal_wave/forward.py

```diff
@@ -8,7 +8,7 @@ class Forward(Step):
     test cases.
     """
     def __init__(self, test_case, name='forward', subdir=None, ntasks=1,
-                 min_tasks=None, openmp_threads=1, nu=None):
+                 min_tasks=None, openmp_threads=1, nu=None, vlr=False):
```

Comment:
This needs to be added to the docstring.

Reply:
Addressed by https://github.com/MPAS-Dev/compass/pull/564/commits/74755efd7497e3937d080aeea63ce0400564cf4d